experimental/dds/tree/src/test/EditLog.tests.ts

```diff
@@ -338,6 +338,53 @@ describe('EditLog', () => {
 		expect(editsEvicted).to.equal(targetEditLogSize * 2 - collaborationWindowSize);
 	});
 
+	it("can handle sparse sequence numbers with a minimum sequence number that's not in memory", () => {
+		const targetEditLogSize = 10;
+		const log = new EditLog(undefined, undefined, undefined, targetEditLogSize, targetEditLogSize * 2);
+		const ids: EditId[] = [];
+		const sequenceNumberInterval = 3;
+
+		let editsEvicted = 0;
+
+		log.registerEditEvictionHandler((editsToEvict) => {
+			editsEvicted += editsToEvict;
+		});
+
+		let sequenceNumber = 0;
+
+		for (let i = 0; i < targetEditLogSize; i++) {
+			const edit = newEdit([]);
+			log.addSequencedEdit(edit, {
+				sequenceNumber,
+				referenceSequenceNumber: sequenceNumber - 1,
+			});
+			sequenceNumber += sequenceNumberInterval;
+			ids.push(edit.id);
+			expect(log.getIndexOfId(edit.id)).equals(i);
+		}
+		expect(log.length)
+			.equals(log.numberOfSequencedEdits)
+			.and.equals(targetEditLogSize, 'Only sequenced edits should be present.');
+
+		const extraEditsToKeep = 3;
+		const collaborationWindowSize = targetEditLogSize + extraEditsToKeep;
+		const minimumSequenceNumber = sequenceNumber + 1 - extraEditsToKeep * sequenceNumberInterval;
```

Comment:
It's adding +1 cause the interval is 3 so the min sequence number is expected to be something like 37 which won't be a number we have stored and so it'd use 36, the number we do have stored, instead. Although, I now realize that we want the higher number instead since 36 wouldn't be part of the collab window. I'll make that change and add a comment.

Reply:
Ah okay, sounds good :)